src/main/openapi/ckan.yaml

```diff
@@ -224,6 +228,16 @@ components:
       required:
         - name
         - email
+    Creator:
+      type: object
+      properties:
+        creator_name:
```

Comment:
i think "name" should be ok instead of "creator_name" , same for create_identifier..